frontend/stores/templates.js

```
@@ -0,0 +1,51 @@
+        return (this.filters = {
+            moreTemplatesAvailable: true,
+            templateJourneyLengthMinMax: [1, TEMPLATE_MAX_LENGTH],
+            PER_PAGE: PER_PAGE,
+            sortby: "",
+            sortorder: "",
+            templateDestinationInput: "",
+            templateDestinationName: "",
+            templateCreator: "",
+            cursor: null,
+            nextCursor: null,
+        });
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)